_sass/code.scss

```diff
@@ -48,17 +50,61 @@ a:visited code {
 
 // ```[LANG]...```
 div.highlighter-rouge,
-div.listingblock > div.content {
-  padding: $sp-3;
+div.listingblock > div.content,
+figure.highlight {
   margin-top: 0;
   margin-bottom: $sp-3;
-  overflow-x: auto;
   background-color: $code-background-color;
   border-radius: $border-radius;
   box-shadow: none;
   -webkit-overflow-scrolling: touch;
+  position: relative;
+  padding: 0;
+
+  > button {
```

Comment:
Could you add a CSS comment explaining what's going on here? I don't think it's immediately obvious to a developer. One or two sentences suffices!

(it's also not immediately clear to me why all of these properties are needed. have you done any pruning on them?)

Reply:
I investigate

Reply:
the only thing I can remove is the height... which solves the problem on mobile